fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -174,17 +171,9 @@
     public static final String RUNTIME_JOIN_FILTER_PUSH_DOWN_LIMIT = "runtime_join_filter_push_down_limit";
     public static final String ENABLE_GLOBAL_RUNTIME_FILTER = "enable_global_runtime_filter";
 
```

Comment:
vectorized_enable_load and enable_vectorized_file_load can also been removed.

Reply:
Removing session variables needs change a lot of UT. I will remove load session variables in next PR